### DISCUSSION TITLE
如何在模型市场添加自己部署的模型信息，目前只能配置模型名称、id、简介等基础信息。哪里有配置模型访问的key，url等信息呢？

### DISCUSSION THREAD

**Discussion by chesonchen** [coaidev/coai#278] [Q&A | 问题解答] [answered]
[//]: # (方框内删除已有的空格，填 x 号)
+ [x] 我已确认目前没有类似 issue
+ [x] 我已确认我已升级到最新版本
+ [x] 我已完整浏览项目 README 和项目文档并未找到解决方案
+ [x] 我理解并愿意跟进此 issue，协助测试和提供反馈
+ [x] 我将以礼貌和尊重的态度提问，不得使用不文明用语 (包括在此发布评论的所有人同样适用, 不遵守的人将被 block)
+ [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**问题描述**
如何在模型市场添加自己部署的模型信息，目前只能配置模型名称、id、简介等基础信息。哪里有配置模型访问的key，url等信息呢？
**复现步骤**
1、使用管理者账号登陆
2、进入后台管理
3、进入模型市场
4、添加模型
**预期结果**
自己添加的模型能够配置好访问信息，普通用户可以使用该模型
**日志信息**

**相关截图 (如果有)**

[a comment by an automated account is not shown]

**Comment by chara006688** [comment, marked as answer]
请在管理面板找到”渠道设置"

[a comment by an automated account is not shown]
